helfomat-core/src/main/java/de/helfenkannjeder/helfomat/core/ProfileRegistry.java

```diff
@@ -17,4 +17,5 @@
 
     public static final String TEST = "test";
 
+    public static final String KAFKA = "kafka";
```

Comment:
Why do you keep in kafka and not just remove it?

Reply:
Because it's easy to keep ;) . Can be removed later.

Reply:
Better remove it now. It makes no sense to keep it.

Reply:
Ok, Just saw that it's merged already ;) make the next PR with removing it ;) Deleted code is good code :)

Reply:
Once the deployment is done and everything is working with postgres.